go.mod

```diff
@@ -0,0 +1,13 @@
+module github.com/rogpeppe/modcop
+
+go 1.12
+
+require (
+	github.com/loggo/loggo v0.0.0-20140128045037-89458b4dc996 // indirect
+	github.com/rogpeppe/go-internal v1.1.0
+	gopkg.in/errgo.v2 v2.1.0
+	launchpad.net/errgo v0.0.0-20140324230230-000000000012
+	launchpad.net/gocheck v0.0.0-20140225173054-000000000087 // indirect
+)
+
+replace github.com/rogpeppe/go-internal => /home/rog/src/go-internal
```

Comment:
I presume this was by mistake! :)

Reply:
Idea: have modcop also complain about replaces pointing outside the VCS repo.

Reply:
The replace is just there until I land https://github.com/rogpeppe/go-internal/pull/51 - I'm holding on for a bit in case you or Paul want to take a look.

modcop will definitely complain about replaces outside the module repo (and perhaps even within the repo if not explicitly whitelisted)

Reply:
removed the replace statement